go/cmd/dolt/commands/send_metrics.go

```diff
@@ -0,0 +1,31 @@
+package commands
+
+import (
+	"context"
+
+	"github.com/liquidata-inc/dolt/go/libraries/doltcore/dbfactory"
+	"github.com/liquidata-inc/dolt/go/libraries/doltcore/env"
+	"github.com/liquidata-inc/dolt/go/libraries/events"
+	"github.com/liquidata-inc/dolt/go/libraries/utils/filesys"
+)
+
+const SendMetricsCommand = "send-metrics"
+
+func SendMetrics(ctx context.Context, commandStr string, args []string, dEnv *env.DoltEnv) int {
+	fs := filesys.LocalFS
+
+	root, err := env.GetCurrentUserHomeDir()
+	if err != nil {
+		// log instead
+		panic(err)
+	}
+
+	dolt := dbfactory.DoltDir
+
+	eventGrpcFlush := events.NewEventGrpcFlush(fs, root, dolt, dEnv)
+	if err := eventGrpcFlush.FlushEvents(); err != nil {
+		// log error?
+		return 1
+	}
+	return 0
+}
```

Comment:
`SendMetrics` is the `commandFunc` for the dolt command "dolt send-metrics". Looking for feedback on how to handle the error logging here. This also gets run in a separate process on all other dolt commands using `os.exec`